artillery.yml

```diff
@@ -0,0 +1,14 @@
+config:
+  target: 'http://localhost:7070'
+  phases:
+    - duration: 60
+      arrivalRate: 5
+scenarios:
+  - name: 'proxy test'
+    flow:
+    - loop:
+      - get:
+          url: '/?{{ $randomNumber(1,1000000) }}'
+          count: 2
```

Comment:
I think the indentation of count is wrong here - have you checked whether this runs as expected?

Reply:
Good point. Fixed!